packages/widget/src/context/wallet.ts

```diff
@@ -14,11 +16,12 @@ export interface EvmWallet {
 }
 
 export interface SubstrateWallet {
-  signer: Signer;
-  signerAddress: string;
-  accounts: Account[];
+  signer?: Signer;
+  signerAddress?: string;
+  accounts?: Account[];
```

Comment:
why does `accounts` have to be `undefined`?

Reply:
because of the way we need to provide `susbtrateProvider` as property that the widget is able to receive. Is either this or initialize by default the arguments of the substrate wallet when `ApiPromise` is created